ui/app/AppLayouts/Wallet/views/collectibles/CollectibleView.qml

```diff
@@ -0,0 +1,88 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import shared.panels 1.0
+
+import "../../stores"
+import utils 1.0
+
+Item {
+    id: root
+    property var collectibleModel
+    width: 176
+    height: 225
+    anchors.bottomMargin: 4
```

Comment:
This should not be set here but from the parent component.

Reply:
Done